scripts/qiita-recover-jobs

```diff
@@ -41,27 +42,38 @@ def _submit_jobs(jids_to_recover, recover_type):
 
 
 def _retrieve_queue_jobs():
-    qiita_jobs = [line.split()[0]
-                  for line in check_output("qstat").decode('ascii').split("\n")
-                  # just retriving 'qiita' and ignoring [] (ipython workers)
-                  if 'qiita' in line and '[]' not in line and
-                  # and private jobs
-                  'private' not in line and
-                  'STDIN' not in line]
+    lines = check_output(["qstat", "-f"]).decode('ascii').split("\n")
```

Comment:
This is minor but it looks like the code in this function could be simplified if the jobs output was parsed into a table with a column for job owner, job name, etc.

Reply:
That's correct and thought about it ... my only concern (and reason I didn't implement like that) is that we will be creating a full parser for the output, which IMOO will add complexity. Note that this is the second iteration of this block and works fine as it is; however, we _might_ need to implement the table suggestion in a following version; just not sure right now.